app/test_main.py

```
@@ -1,3 +1,37 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        # Values for tests
+        (20, 7, [1, 0]),
+        (15.1, 15.1, [1, 1]),
```

This violates checklist item: "Non-integer types: verify that passing incorrect types raises the correct exception (typically TypeError or ValueError; tests must assert a specific exception)." The test uses float values (15.1, 15.1) as valid input, but the requirements specify that only integers are allowed and non-integer types should raise an exception. You should replace this with a test that asserts a TypeError is raised for float input.